spec/helpers/admin/departments_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the Admin::DepartmentsHelper. For example:
+#
+# describe Admin::DepartmentsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe Admin::DepartmentsHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
You can try as like:
context 'this test only passes' do
it 'the simple test code' do
    expect(2+2).to eq(4)
  end
end